<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonGO">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>200</width>
      <height>63</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>GO!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDataCompare">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Проверка записанных данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConsole">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>78</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditConsole">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>125</y>
      <width>581</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>98</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLockChip">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Защита FLASH памяти и EEPROM от копирования</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pictures/help.png</normaloff>:/pictures/help.png</iconset>
   </property>
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
